invenio_rdm_records/services/config.py

```diff
@@ -210,6 +210,26 @@ class RDMSearchOptions(SearchOptions, SearchOptionsMixin):
     ]
 
 
+class RDMCommunityRecordSearchOptions(SearchOptions, SearchOptionsMixin):
+    """Search options for community record search."""
+
+    facets = {
+        "resource_type": facets.resource_type,
+        "languages": facets.language,
+        "access_status": facets.access_status,
+    }
+
+    # without VerifiedRecordsSortParam as communities review records they accept
+    params_interpreters_cls = [
+        AllVersionsParam.factory("versions.is_latest"),
+        QueryStrParam,
+        PaginationParam,
+        FacetsParam,
+        StatusParam,
+        PublishedRecordsParam,
+    ] + SearchOptions.params_interpreters_cls
```

Comment:
```suggestion
    ]
```

This would already include many of the other parameters already specified, and I don't want to know what happens when we repeat these classes 😅 

I think now that we've identified where the issue was, I would actually go another solution, which is to keep the explicit `RDMCommunityRecordSearchOptions`, but modify `VerifiedRecordsSortParam` so that it checks for `self.config.verified_sorting_enabled`, which we can then set like:

```python
class RDMCommunityRecordSearchOptions(...):

   verified_sorting_enabled = False
```

Thus, we keep the original records search config via inheritance and don't have to duplicate new facets or parameters.